README.md

```diff
@@ -381,7 +381,7 @@ myNextApplication:
       apiLambda: "nodejs10.x"
 ```
 
-Similarly, the timeout by default is 10 seconds. To customise you can:
+Similarly, the timeout by default is 10 seconds. To customize you can:
```

Comment:
Thanks for your contribution! This is actually not a typo, as the original author of this package @danielcondemarin is based in the UK 😛  and these are the British spellings.

I'm fine with reorganizing the images though, since there are a lot of files in the repository root.